### PR TITLE
Negative Margins

### PR DESCRIPTION
**Pull request by XinYenFon**
I still have no idea why my git client deletes some } and re-adds it but (investigating - still better than my win-git deleting everything and adding back)

### CODE REVIEW

**Comment by jdarwood007** [comment]
The end of file can occur because some editors add a new line if one doesn't exist. Such as using nano command line well do this.

-Sent from my Android

File: Themes/default/css/index.css [hunk 1]
```diff
@@ -1680,7 +1680,7 @@ img#upshrink {
 	padding: 0 0 0 6px;
 }
 .mark_read {
-	margin: -5px 0 0 0;
+	margin: 0 0 0 0;
```
Comment (live627) [inline Themes/default/css/index.css:1683]: It can be futhur simplified to one zero instead of four.

**Comment by XinYenFon** [comment]
@live627 
Yea, but there is lot of 0 0 0 0 in index.css and for now i don't want to mess with it.

@jdarwood007 
Any suggestion ? (Using Ubuntu 12.04 - Default Text Editor)

**Comment by ghost** [comment]
Why do you want to change it? What is wrong with it the way it is?

**Comment by XinYenFon** [comment]
http://i49.tinypic.com/29w91sh.png 

In default install -5px on mark_read makes button misplaced

Edit : This happens with Opera & Chromium

**Comment by ghost** [comment]
Don't worry about it. Test with more boards. That was due to the divider td having no height set, which meant it collapsed in Opera and Webkit (since they wont accept a height on the tbody instead) which meant there was no spacing after each catgeory. It's fixed. Ignore it.

ETA: Take a look at the new posts, etc icons and text too. They're obviously wrong, for the same reason. The button is correctly aligned with the icons and text.

**Comment by ghost** [comment]
I started the theme in Firefox and have not done a lot of testing in other browsers yet. It's not a problem, because the browsers we are supporting are all very close to standards compliance, which means any buglets are easy to fix.

Please just ignore any apparent theme bugs until after the weeked.

**Pull request closed without merging**